services/order.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+from db.models import Ticket, Order, MovieSession
+
+
+def create_order(tickets: list,
+                 username: str,
+                 date: datetime = None) -> Ticket:
+    with transaction.atomic():
```

Comment:
Consider using the `@transaction.atomic` decorator instead of the `with transaction.atomic():` block for better readability and consistency with Django's transaction management practices.